src/App.vue

```diff
@@ -9,13 +9,17 @@
             <HTMLGenerator :shortLicenseName="selected.shortName"/>
         </div>
         <Footer>
-            <p>
-                <a href="https://github.com/creativecommons/cc-chooser">Icons</a> by The Noun Project and FontAwesome.
-            </p>
-            <p>
-                View this page's source code <a href="https://github.com/creativecommons/cc-chooser">on Github</a>.
```

Comment:
Please add this link back into the footer, that’s the only thing I can see that I would like modified! I’ll take a look on my computer when I get home but this should be the last change before merge :)

Reply:
To be clear, I mean the “View this page’s source code” link

Reply:
Is the latest commit okay? Can I remove WIP?

Reply:
Just realized, please remove the wikimedia commons link and I'll get this merged! Thanks :)

Reply:
Done, thanks.